app/marketing/mails.py

```diff
@@ -35,10 +37,14 @@
     render_start_work_approved, render_start_work_new_applicant, render_start_work_rejected, render_tip_email,
 )
 from sendgrid.helpers.mail import Content, Email, Mail, Personalization
+from sendgrid.helpers.stats import Category
+
 
+def func_name():
+    return inspect.stack()[0][3]
```

Comment:
from my testing... i never seemed to generate an exception... what exception are you worryed about?

either `inspect.*` option works for me...

Reply:
I only say that because we're running `inspect` in a production environment.  If we ever opted to move away from CPython for whatever reason, it would bomb out.  Also, I mention `inspect.currentframe().f_code.co_name` versus `inspect.stack()[0][3]` because it's typically **MUCH** faster to run the currentframe call over stack.